### ISSUE TITLE
[Deque Analysis] Reorder buttons in Edit Item / Bitstreams forms are not keyboard accessible (Critical Issue) 

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#3154]
> Critical: This issue results in blocked content for people with disabilities, and will definitely prevent them from accessing fundamental features or content. This type of issue puts your organization at risk. Prioritize fixing as soon as possible, within the week if possible. Remediation should be a top priority.

## Deque Analysis Summary

The buttons used to reorder multi-valued fields, or Bitstreams within an item are not fully keyboard accessible. Currently, while you can tab to the button, you cannot "click" it by pressing Enter.  

(Issue ID: 470936) Edit Item, Bitstreams tab: Button to reorder bitstreams (in a bundle) is not keyboard accessible.

(NOTE: This is related to #1271, which is a similar issue related to reordering fields in the submission form.)

The full list of these issues (which includes links for more info) can be found at this link (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues?activeTab=dt-issue&page=0&pageSize=100&sortField=ordinal&sortDir=asc&row=9&filter%5Bseverity%5D=4&filter%5Bpage_number%5D=12&filter%5Bpage_number%5D=13&filter%5Bpage_number%5D=14&filter%5Bpage_number%5D=15

## Possible solutions
As mentioned in https://github.com/DSpace/dspace-angular/issues/1271#issuecomment-965575328 

> ... we may want to consider whether there's a way to more easily make these reorder buttons more accessible by using arrow keys to reorder the list. Here's an example of what I'm talking about with regards to using arrow keys to sort a list of items: https://medium.com/salesforce-ux/4-major-patterns-for-accessible-drag-and-drop-1d43f64ebf09#0303 (And another example of the same idea: https://www.barrierbreak.com/can-drag-and-drop-be-made-accessible-yes-it-can-be-in-different-ways/)

## More Information / Tools

* Related to #1271 and may be solved in a similar way.
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work

**Comment by artlowel** [comment]
Atmire would like to claim this issue as well as https://github.com/DSpace/dspace-angular/issues/1270 and https://github.com/DSpace/dspace-angular/issues/1191, and refactor this page to solve them all

**Comment by artlowel** [comment]
@tdonohue This will take about 24hrs
